Sources/Plasma/FeatureLib/pfConsoleCore/pfConsoleCmd.h

```diff
@@ -282,4 +284,10 @@ class pfConsoleCmdIterator
 #define PF_CONSOLE_FILE_DUMMY( name ) \
     void _console_##name##_file_dummy() { }
 
+template <typename... Args>
+void pfConsolePrintF(void pfun(const char *), const char *fmt, Args &&...args)
```

Comment:
Style nit (pointer binds to the type, not the argument)

Reply:
I tend to follow the prevailing style in the modified file for pointer/reference alignment...  In this case, the vast majority of the file binds to the name, not the type.

Reply:
I guess we can save the formatting fixes for later in that case.